### PR TITLE
chore(RHTAPWATCH-527): Update readme file

### PR DESCRIPTION
**Pull request by avi-biton**
Update readme file after adding control plane rules to the repo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,11 +32,12 @@ The different alerting rules in this repository are:
...
-### Updating Data Plane Alerts
+### Updating Alerts
 
-Alert rules for data plane clusters are being deployed by app-interface to RHOBS, to where the data plane metrics are also being forwarded. For deploying the alert rules,
-app-interface references the location of the rules together with a git reference -
-branch name or commit hash.
+Alert rules for data plane and control plane clusters are being deployed by app-interface 
+to RHOBS, to where the metrics are also being forwarded. For deploying the 
+alert rules, app-interface references the location of the rules together with a git 
+reference -branch name or commit hash.
```
Comment (yftacherzog) [inline README.md:40]:
```suggestion
reference - branch name or commit hash.
```

File: README.md [hunk 2]
```diff
@@ -50,10 +48,12 @@ Steps for updating the rules:
 
 1. Merge the necessary changes to this repository - alerts and tests.
 2. Update the
-[staging environment](https://gitlab.cee.redhat.com/service/app-interface/-/blob/0486ef164e70259e5b85c46ab749529238368414/data/services/osd-operators/cicd/saas/saas-rhtap-rules.yaml#L35)
+[data plane staging environment](https://gitlab.cee.redhat.com/service/app-interface/-/blob/2a88da3b5ed7589a3954ebb31c484702ef49cd91/data/services/osd-operators/cicd/saas/saas-rhtap-rules.yaml#L35) and/or the
+[control plane staging environment](https://gitlab.cee.redhat.com/service/app-interface/-/blob/2a88da3b5ed7589a3954ebb31c484702ef49cd91/data/services/osd-operators/cicd/saas/saas-rhtap-rules.yaml#L49)
```
Comment (yftacherzog) [inline README.md:52]:
```suggestion
[data plane staging environment](https://gitlab.cee.redhat.com/service/app-interface/-/blob/2a88da3b5ed7589a3954ebb31c484702ef49cd91/data/services/osd-operators/cicd/saas/saas-rhtap-rules.yaml#L35)
and/or the
[control plane staging environment](https://gitlab.cee.redhat.com/service/app-interface/-/blob/2a88da3b5ed7589a3954ebb31c484702ef49cd91/data/services/osd-operators/cicd/saas/saas-rhtap-rules.yaml#L49)
```

File: README.md [hunk 3]
```diff
@@ -50,10 +48,12 @@ Steps for updating the rules:
...
 reference in app-interface to the commit hash of the changes you made.
 3. Once merged and ready to be promoted to production, update the
-[production environment](https://gitlab.cee.redhat.com/service/app-interface/-/blob/0486ef164e70259e5b85c46ab749529238368414/data/services/osd-operators/cicd/saas/saas-rhtap-rules.yaml#L39) reference in a similar manner.
+[data plane production environment](https://gitlab.cee.redhat.com/service/app-interface/-/blob/2a88da3b5ed7589a3954ebb31c484702ef49cd91/data/services/osd-operators/cicd/saas/saas-rhtap-rules.yaml#L39) and/or the
+[control plane production environment](https://gitlab.cee.redhat.com/service/app-interface/-/blob/2a88da3b5ed7589a3954ebb31c484702ef49cd91/data/services/osd-operators/cicd/saas/saas-rhtap-rules.yaml#L53) reference in a similar manner.
```
Comment (yftacherzog) [inline README.md:56]:
```suggestion
[data plane production environment](https://gitlab.cee.redhat.com/service/app-interface/-/blob/2a88da3b5ed7589a3954ebb31c484702ef49cd91/data/services/osd-operators/cicd/saas/saas-rhtap-rules.yaml#L39)
 and/or the
[control plane production environment](https://gitlab.cee.redhat.com/service/app-interface/-/blob/2a88da3b5ed7589a3954ebb31c484702ef49cd91/data/services/osd-operators/cicd/saas/saas-rhtap-rules.yaml#L53)
 reference in a similar manner.
```

File: README.md (hunk 1 above)
Comment (avi-biton) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (avi-biton) [reply README.md]: Done

File: README.md (hunk 3 above)
Comment (avi-biton) [reply README.md]: Done

**Review by yftacherzog — APPROVED** [review]

**Review by ifireball — APPROVED** [review]
LGTM

**Pull request merged**